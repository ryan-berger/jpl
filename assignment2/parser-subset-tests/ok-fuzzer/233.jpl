write image .92452 to ""
show Z()
return P
show h
read image "" to J
read image "p" to b
assert t7B_3Z.(C), "(by"
print ""
write image Y to ""
time read image "" to q3x_
return P9h_()
show .78
time read image "" to w3b_
print ""
print ""
let u = 35.
print ""
time write image 9. to "P"
time time write image 27 to ""
