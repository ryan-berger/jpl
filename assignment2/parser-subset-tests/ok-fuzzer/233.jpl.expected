(WriteImageCmd (FloatExpr 0) "")
(ShowCmd (CallExpr Z))
(StmtCmd (ReturnStmt (VarExpr P)))
(ShowCmd (VarExpr h))
(ReadImageCmd "" (VarArgument J))
(ReadImageCmd "p" (VarArgument b))
(StmtCmd (AssertStmt (CallExpr t7B_3Z. (VarExpr C)) "(by"))
(PrintCmd "")
(WriteImageCmd (VarExpr Y) "")
(TimeCmd (ReadImageCmd "" (VarArgument q3x_)))
(StmtCmd (ReturnStmt (CallExpr P9h_)))
(ShowCmd (FloatExpr 0))
(TimeCmd (ReadImageCmd "" (VarArgument w3b_)))
(PrintCmd "")
(PrintCmd "")
(StmtCmd (LetStmt (ArgLValue (VarArgument u)) (FloatExpr 35)))
(PrintCmd "")
(TimeCmd (WriteImageCmd (FloatExpr 9) "P"))
(TimeCmd (TimeCmd (WriteImageCmd (IntExpr 27) "")))
